device-types/MikroTik/RB951Ui-2nD,yaml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: RB951Ui-2nD-tst
+slug: rb951ui-2nd-tst
```

Comment:
Please append `mikrotik-` to the slug
The full slug for this should be `mikrotik-hap`

Reply:
done